AltexSoft lab 1/scss/style.scss

```diff
@@ -0,0 +1,782 @@
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    padding: 18px 20px;
+    top: 0;
+    background-color: inherit;
+    z-index: 90;
+
+    &.is-scroll {
+      background-color: $color-white;
+    }
+
+    .logo {
+      width: $size-logo-mobile;
+    }
+
+    #logo.is-scroll {
+      display: none;
+    }
+
+    #logo-dark {
+      display: none;
+
+      &.is-scroll {
+        display: block;
+      }
+    }
+
+    .logo-dark {
+      width: $size-logo-mobile;
+    }
+
+    .menu-btn {
+      position: relative;
+      width: 24px;
+      height: 18px;
+      border: none;
+      background: transparent;
+      z-index: 99;
+
+      &.is-scroll > span {
+        background-color: $color-primary-dark;
+      }
+
+      &.is-clicked {
+        .menu-icon {
+          &:first-child {
+            top: 50%;
+            transform: rotate(-45deg) translate(0px, -80%);
+          }
+
+          &:nth-child(2) {
+            opacity: 0;
+          }
+
+          &:last-child {
+            bottom: 50%;
+            transform: rotate(45deg) translate(0px, 50%);
+          }
+        }
+      }
+
+      .menu-icon {
+        position: absolute;
+        width: 24px;
+        height: 2px;
+        right: 0;
+        background-color: $color-white;
+        transition: .4s;
+
+        &:first-child {
+          top: 0;
+        }
+
+        &:nth-child(2) {
+          top: 8px;
+        }
+
+        &:last-child {
+          bottom: 0;
+        }
+      }
+    }
+
+    .nav-links {
+      position: fixed;
+      display: none;
+      width: 100%;
+      height: 100vh;
+      top: 61px;
+      left: 0;
+      background-color: $color-white;
+      z-index: 98;
+
+      .menu {
+        padding-top: 40px;
+        text-align: center;
+        list-style: none;
+        color: $color-clicable-el;
+
+        .menu-item {
+          margin-bottom: 16px;
+
+          .nav-link {
+            @include letters-xs;
+
+            text-decoration: none;
+            font-family: $font-family-acme;
+            letter-spacing: 2px;
+            text-transform: uppercase;
+            color: inherit;
+
+            &.active {
+              border-bottom: 2px solid $color-clicable-el;
+            }
+          }
+        }
+      }
+
+      .menu-get-skout-btn,
+      .menu-login-btn {
+        @include letters-xs;
+
+        margin: 0 auto;
+        border: 1.5px solid $color-clicable-el;
+        border-radius: 20px;
+        font-family: $font-family-acme;
+        letter-spacing: 2px;
+        text-transform: uppercase;
+        color: $color-clicable-el;
+        cursor: pointer;
+        background-color: transparent;
+
+        &:hover {
+          background-color: $color-clicable-el;
+          color: $color-white;
+          transition: 0.7s;
+        }
+      }
+
+      .menu-get-skout-btn {
+        display: block;
+        padding: 7px 54px;
+        margin-top: 205px;
+      }
```

Comment:
![image](https://user-images.githubusercontent.com/84564899/121024099-94390800-c7ac-11eb-85e3-6de0121e55f8.png)
the number of links can be different and these buttons will be invisible due to hardcoding of this indentation